MobSF/settings.py

```diff
@@ -292,7 +294,7 @@
     JD_CORE_DECOMPILER_BINARY = ""
     PROCYON_DECOMPILER_BINARY = ""
     APKTOOL_BINARY = ""
-    ADB_BINARY = ""
+    ADB_BINARY = "adb"

```

Comment:
How about defaulting to appropriate version from `/DynamicAnalyzer/tools/adb/` ?